lib/bolt/plugin.rb

```diff
@@ -36,6 +36,12 @@ def initialize(plugin_name, hook)
           super("Plugin #{plugin_name} does not support #{hook}", 'bolt/unsupported-hook')
         end
       end
+
+      class PluginLoadingDisabled < PluginError
```

Comment:
I feel like `LoadingDisabled` would feel weird b/c the name does not explicitly tell me _what_ is being loaded. Hence, I feel like `PluginLoadingDisabled` may be the better name here.